docs/pages/versions/unversioned/sdk/facebook.md

```diff
@@ -161,19 +161,19 @@ Given a valid Facebook application ID in place of `<APP_ID>`, the code above wil
 
 Logs out of the currently authenticated session.
 
-### `Facebook.getUserAuthAsync()`
+### `Facebook.getCredentialStateAsync()`
```

Comment:
This should be `getCredentialAsync()`. In the Apple auth library, `AppleAuthenticationCredentialState` is a separate type from `AppleAuthenticationCredential` and [`getCredentialStateAsync()` returns a CredentialState](https://docs.expo.io/versions/latest/sdk/apple-authentication/#appleauthenticationgetcredentialstateasyncuser).

```suggestion
### `Facebook.getCredentialAsync()`
```